util/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/DicomClient.java

```diff
@@ -51,18 +61,57 @@ public static DicomClient associatePeer(
     return new DicomClient(association);
   }
 
+  public static void cstore(
+      String sopClassUid,
+      String sopInstanceUid,
+      String transferSyntaxUid,
+      InputStream in,
+      ApplicationEntity applicationEntity,
+      String dimsePeerAet,
+      String dimsePeerHost,
+      int dimsePeerPort) throws IOException, InterruptedException {
+    PresentationContext pc = new PresentationContext(1, sopClassUid, transferSyntaxUid);
+    DicomClient dicomClient;
+    try {
+      dicomClient = DicomClient.associatePeer(applicationEntity,
+          dimsePeerAet, dimsePeerHost, dimsePeerPort, pc);
+    } catch (IOException | IncompatibleConnectionException | GeneralSecurityException e) {
+      throw new IOException(e);
```

Comment:
Because calling code doesn't need to distinguish between these at current point (1st pull request contains only one of 2 callers, the other one is in c-move).
I could change callers to catch more exceptions if you think that would be better.

Reply:
Ok